test/JDBC/replication/partition-replication.mix

```diff
@@ -0,0 +1,1481 @@
+GO
+
+-- Insert values into partition_vu_prepare_smallmoney_table
+INSERT INTO partition_vu_prepare_smallmoney_table (Id, Value) VALUES (4.91, 'Value 4.91');
+GO
+
+-- Insert values into partition_vu_prepare_date_table
+INSERT INTO partition_vu_prepare_date_table (Id, Value) VALUES ('2023-01-01', 'Value 2023-01-01');
+GO
+
+-- Insert values into partition_vu_prepare_datetime_table
+INSERT INTO partition_vu_prepare_datetime_table (Id, Value) VALUES ('2022-07-01', 'Value 2022-07-01');
+GO
+
+-- Insert values into partition_vu_prepare_datetime2_table
+INSERT INTO partition_vu_prepare_datetime2_table (Id, Value) VALUES ('2022-07-01 00:00:00.000', 'Value 2022-07-01 00:00:00.000');
+GO
+
+-- Insert values into partition_vu_prepare_smalldatetime_table
+INSERT INTO partition_vu_prepare_smalldatetime_table (Id, Value) VALUES ('2022-07-01', 'Value 2022-07-01');
+GO
+
+-- Insert values into partition_vu_prepare_char_table
+INSERT INTO partition_vu_prepare_char_table (Id, Value) VALUES ('U', 'Value U');
+INSERT INTO partition_vu_prepare_char_table (Id, Value) VALUES ('D', 'Value D');
+GO
+
+-- Insert values into partition_vu_prepare_varchar_table
+INSERT INTO partition_vu_prepare_varchar_table (Id, Value) VALUES ('Mango', 'Value Mango');
+INSERT INTO partition_vu_prepare_varchar_table (Id, Value) VALUES ('Cherry', 'Value Cherry');
+INSERT INTO partition_vu_prepare_varchar_table (Id, Value) VALUES ('Date', 'Value Date');
+GO
+
+-- Insert values into partition_vu_prepare_nvarchar_table
+INSERT INTO partition_vu_prepare_nvarchar_table (Id, Value) VALUES (N'Mango', N'Value Mango');
+INSERT INTO partition_vu_prepare_nvarchar_table (Id, Value) VALUES (N'Cherry', N'Value Cherry');
+INSERT INTO partition_vu_prepare_nvarchar_table (Id, Value) VALUES (N'Date', N'Value Date');
+GO
+
+-- Insert values into partition_vu_prepare_nchar_table
+INSERT INTO partition_vu_prepare_nchar_table (Id, Value) VALUES (N'U', N'Value U');
+INSERT INTO partition_vu_prepare_nchar_table (Id, Value) VALUES (N'B', N'Value B');
+GO
+
+-- Insert values into partition_vu_prepare_binary_table
+INSERT INTO partition_vu_prepare_binary_table (Id, Value) VALUES (0x0800, 'Value 0x0800');
+INSERT INTO partition_vu_prepare_binary_table (Id, Value) VALUES (0x0C000, 'Value 0x0C000');
+GO
+
+-- Insert values into partition_vu_prepare_varbinary_table
+INSERT INTO partition_vu_prepare_varbinary_table (Id, Value) VALUES (0x8000000000000000, 'Value 0x8000000000000000');
+INSERT INTO partition_vu_prepare_varbinary_table (Id, Value) VALUES (0xC000000000000000, 'Value 0xC000000000000000');
+GO
+
+-- Insert values into partition_vu_prepare_uniqueidentifier_table
+INSERT INTO partition_vu_prepare_uniqueidentifier_table (Id, Value) VALUES ('FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF', 'Value FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF');
+GO
+
+
+-- Wait for data to get replicated
+-- psql
+select pg_sleep(10);
+GO
+
+-----------------------------------------------------------
+-- Validate that new data's are also replicated
```

Comment:
Nit: That `'s` doesn't look right with data and `data` is a singular entity so we should use `data is` instead of `data are`.